src/backend/core/api/serializers.py

```diff
@@ -26,22 +26,69 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
-class UserSerializer(serializers.ModelSerializer):
+class UserCompleteSerializer(serializers.ModelSerializer):
     """Serialize users."""
 
     timezone = TimeZoneSerializerField(use_pytz=False, required=True)
+    name = serializers.SerializerMethodField(read_only=True)
+    email = serializers.SerializerMethodField(read_only=True)
 
     class Meta:
         model = models.User
         fields = [
             "id",
+            "name",
             "email",
             "language",
             "timezone",
             "is_device",
             "is_staff",
```

Comment:
@lebaudantoine We only need the first 3 fields for the teamaccess view. I chose to return only the necessary info and have a dedicated, shorter serializer (UserSerializer).

EDIT : I've found a way to have a single UserSerializer, but filter fields outside when calling the serializer. I'm pushing the code right away.